### PR TITLE
inital map PR

### PR DESCRIPTION
**Pull request by akadouri**

### CODE REVIEW

File: index.html
Comment (akadouri) [inline index.html:1]: I ran https://prettier.io/ on this to format it

File: index.html [hunk 1]
```diff
@@ -1,51 +1,261 @@
...
+            href="https://www.ridersalliance.org/events/emergency-rally-broadway-junction"
+            >Rally with Riders Alliance</a
+          >
+        </li>
+      </ul>
+    </div>
+    <div id="map"></div>
+  </body>
+  <script>
+    const MAPTILER_KEY = "KmFaB2G58GxhXYCZaw1u";
```
Comment (akadouri) [inline index.html]: You'll need to make a website specific key on https://www.maptiler.com/ to get the background map to load, I can do this in an hour or two

File: index.html [hunk 2]
```diff
@@ -1,51 +1,261 @@
...
+      center: [-74.0066, 40.7135],
+      zoom: 8.5,
+      container: "map",
+      antialias: true,
+    });
+    map.on("load", () => {
+      map.addSource("senate", {
+        type: "geojson",
+        // Use a URL for the value for the `data` property.
+        data: "http://localhost:3000/map/updated_Senate.geojson",
```
Comment (akadouri) [inline index.html]: will have to update this URL as well for the published link

File: map/updated_Senate.geojson
Comment (akadouri) [inline map/updated_Senate.geojson:1]: I have to cleanup the script to generate this a little bit more, it has my google keys in it.

File: index.html (hunk 1 above)
Comment (geofft) [reply index.html]: I made a key and restricted it to savecongestionpricing.org, thanks! I'll update this PR with it.

File: index.html
Comment (geofft) [reply index.html:1]: I'm going to be annoying and undo this to make the diff more readable, but I probably should reformat it. I'm doing a fair amount of editing from my phone though so I'll probably optimize for what makes that easiest.

(... which might be generating from Markdown, tbh)

File: map/updated_Senate.geojson
Comment (geofft) [reply map/updated_Senate.geojson:1]: I have GitHub Actions set up to do a generation step for this, currently it's just doing a `cp` but we can add a script and have it use secret environment variables

File: index.html (hunk 2 above)
Comment (geofft) [reply index.html]: I think it can just be relative

**Comment by geofft** [comment]
Thank you! I'm merging your code as a separate file so we can tweak it without affecting the live site quite yet.

File: index.html
Comment (akadouri) [reply index.html:1]: no problem, there was a missing `</body>` tag that I couldn't find until I reformatted it.

**Pull request merged**